www-src/app/replicator/replication_launcher.coffee

```diff
@@ -88,6 +90,7 @@ module.exports = class ReplicationLauncher
         if options.live
             liveOptions =
               retry: true
+              heartbeat: false
```

Comment:
It's still broken (PouchDB nunkown error) when I debug with my cozy-dev . So I removed heartbeat to remove noise while debuggin, but I haven't yet try with a prod cozy, to check if it works as well as kossi experience it.
Just a minute and I will remove it from this PR : )